File: framework/cei_python3/impl/websocketmessage.py
class WebSocketMessage:

    def __init__(self, text: str = str(), byte_s: bytes = bytes()):
        self.__text = text
        self.__bytes = byte_s

    def get_string(self):
        return self.__text

    def get_bytes(self):
        return self.__bytes

    def upgrade(self, value: str):
        self.__text = value

File: framework/cei_golang/src/impl/websocketmessage.go
package impl

type WebSocketMessage struct {
	text      string
	byteValue []byte
}

func NewWebSocketMessageByString(text string) *WebSocketMessage {
	msg := new(WebSocketMessage)
	msg.text = text
	return msg
}

func (inst *WebSocketMessage) GetString() string {
	return inst.text
}

func (inst *WebSocketMessage) GetBytes() []byte {
	return inst.byteValue
}

func (inst *WebSocketMessage) Upgrade(value string) {
	inst.text = value
}
